### PR TITLE
Compatibility for Django 1.9

### PR DESCRIPTION
**Pull request by Gaffney**
@terite

### CODE REVIEW

File: impersonate/urls.py [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
+from django.conf.urls import url
 
-urlpatterns = patterns(
-    'impersonate.views',
-    url(r'^stop/$',
-        'stop_impersonate',
-        name='impersonate-stop'),
-    url(r'^list/$',
-        'list_users',
-        {'template': 'impersonate/list_users.html'},
```
Comment (terite) [inline impersonate/urls.py]: what happened to the custom template support?

**Comment by terite** [comment]
It looks like changes to `tox.ini` are missing.

@Gaffney Are you up to adding support for automated CI builds? I think these changes are fine (except for the backwards compatibility template break), but would like to easily see a green build matrix to be sure.

**Comment by Gaffney** [comment]
@terite I am not a repo admin, so it looks like I can't do that.

add_projects_-_circleci

**Comment by terite** [comment]
@Gaffney Whoops! I just enabled travis-ci for this repository.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - TOXENV=py3.5-django1.9
+    - TOXENV=py3.4-django1.9
+    - TOXENV=py3.3-django1.9
+    - TOXENV=py2.7-django1.9
+    - TOXENV=py2.6-django1.9
+
+matrix:
+    include:
+        - python: "2.7"
+          env: TOXENV=flake8
```
Comment (terite) [inline .travis.yml]: in `tox.ini`, there are no toxenvs for flake8 or coverage

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: python
+
+python:
+    - "2.7"
+    - "3.4"
+
+env:
+    - TOXENV=django19
```
Comment (terite) [inline .travis.yml]: Does this only support Django 1.9? I think it'd be a good idea to make sure this still supports 1.8

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+sudo: false
+language: python
+
+python:
+    - "2.7"
+    - "3.4"
```
Comment (terite) [inline .travis.yml]: Testing against python 3.5 would be good too

**Comment by Gaffney** [comment]
Utter waste of time, closing in favor of submitting PRs to the upstream and not relying on maintaining an outdated fork.

**Pull request closed without merging**